config/config.yaml

```diff
@@ -1,7 +1,7 @@
 global:
-  DATA_PATH: "./data/"
-  ACCOUNTS_PATH: "./.accounts/"
-  LOCAL_VSTORE_PATH: "./data/vstore/"
+  DATA_PATH: "/root/data/"
```

Comment:
Is there a better way to have the folders set? I'm worried on a blank install, it's not obvious that the person installing A2rchi has this `root` directory. However, I know that we do need to make it work for our install on the service account.

Reply:
So this `/root` directory is guaranteed to exist because the Dockerfiles used by Docker compose are built on the `python:3.10` image, which has this path. If we eventually support a BYOC ("bring your own container") deployment then we will need to change this, but that should happen in its own PR so I think this is fine for now.